### PR TITLE
[4050] Add data sharing agreement page and link from footer

### PR DESCRIPTION
**Pull request by AbigailMcP**
### Context

https://trello.com/c/PDOYLnAN/4050-dsa-upload-to-production

### Changes proposed in this pull request

- Add the data sharing agreement page
- Linked from footer

### Important business

- [ ] Does this PR introduce any PII fields that need to be overwritten or deleted in db/scripts/sanitise.sql?
- [ ] Does this PR change the database schema? If so, have you updated the config/analytics.yml file and considered whether you need to send 'import_entity' events?

NB: Please notify the #twd_data_insights team and ask for a review if new fields are being added to analytics.yml

### CODE REVIEW

**Comment by AbigailMcP** [comment]
[DSA Upload to production](https://trello.com/c/PDOYLnAN/4050-dsa-upload-to-production)

**Comment by AbigailMcP** [comment]
[DSA Upload to production](https://trello.com/c/PDOYLnAN/4050-dsa-upload-to-production)

File: config/locales/pages/data_sharing_agreement/en.yml [hunk 1]
```diff
@@ -0,0 +1,567 @@
...
+        </h2>
+        <p class='govuk-body'>
+          Refer to the <a class='govuk-link' href='/privacy-policy'></a>Register
+          privacy notice</a> to see how Register handles data entered
+          directly into the service.
+        </p>
+        <p class='govuk-body'>
+          In a situation where trainees’ records are entered directly
+          into Register for (non Apply trainees), the provider should
+          inform their trainees of Register’s own privacy policy which
```
Comment (HettieS) [inline config/locales/pages/data_sharing_agreement/en.yml:239]:
```suggestion
          inform their trainees of Register’s own privacy notice which
```

File: config/locales/pages/data_sharing_agreement/en.yml [hunk 2]
```diff
@@ -0,0 +1,567 @@
...
+            in the event of a significant security breach or other
+            serious breach of the terms of this DSA by either
+            participant, the DSA will be terminated or suspended
+            immediately without notice
+          </li>
+        </ul>
+        <h2 class='govuk-heading-m' id='#your-consent'>
+          Your consent
+        </h2>
+        <p class='govuk'>
```
Comment (HettieS) [inline config/locales/pages/data_sharing_agreement/en.yml:562]: I think maybe this class should be `class='govuk-body'` as formatting looks a little weird locally.

Comment (HettieS) [reply config/locales/pages/data_sharing_agreement/en.yml:562]: Screenshot 2022-04-27 at 14 48 40

**Review by HettieS — COMMENTED** [review]
Just a couple comments I've added for correction!

**Review by HettieS — APPROVED** [review]

File: config/locales/pages/data_sharing_agreement/en.yml [hunk 3]
```diff
@@ -0,0 +1,571 @@
...
+            Register retrieves course information from the Publish
+            Teacher Training Service
+          </li>
+          <li>
+            School administrators connect to Register to add, view or
+            update their trainee data. Users can only access data
+            connected to their organisation
+          </li>
+          <li>
+            Register reads and writes rainee record data to systems
```
Comment (MaeveRoseveare) [inline config/locales/pages/data_sharing_agreement/en.yml:294]:
```suggestion
            Register reads and writes trainee record data to systems
```

**Review by MaeveRoseveare — APPROVED** [review]
Just spotted a small typo!

**Pull request merged**